<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>move_clock::SettingsDialog</class>
 <widget class="QDialog" name="move_clock::SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Move Clock Plugin Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="x_group_box">
     <property name="title">
      <string notr="true">x(t)=x0+kt+a sin(nt)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="x0_label">
        <property name="text">
         <string notr="true">x0:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="x0_value"/>
      </item>
      <item>
       <widget class="QLabel" name="kx_label">
        <property name="text">
         <string notr="true">k:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="kx_value"/>
      </item>
      <item>
       <widget class="QLabel" name="ax_label">
        <property name="text">
         <string notr="true">a:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ax_value"/>
      </item>
      <item>
       <widget class="QLabel" name="nx_label">
        <property name="text">
         <string notr="true">n:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="nx_value"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="y_group_box">
     <property name="title">
      <string notr="true">y(t)=y0+kt+a cos(nt)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="y0_label">
        <property name="text">
         <string notr="true">y0:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="y0_value"/>
      </item>
      <item>
       <widget class="QLabel" name="ky_label">
        <property name="text">
         <string notr="true">k:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ky_value"/>
      </item>
      <item>
       <widget class="QLabel" name="ay_label">
        <property name="text">
         <string notr="true">a:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ay_value"/>
      </item>
      <item>
       <widget class="QLabel" name="ny_label">
        <property name="text">
         <string notr="true">n:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ny_value"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="speed_h_layout">
     <item>
      <widget class="QLabel" name="speed_label">
       <property name="text">
        <string>speed:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="speed_slider">
       <property name="minimum">
        <number>25</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>move_clock::SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>move_clock::SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
